frontend/src/App.js

```diff
@@ -72,6 +75,20 @@ function App() {
               <Route exact path="/ucsbdates/create" element={<UCSBDatesCreatePage />} />
             </>
           )
+        }
+             {
+          hasRole(currentUser, "ROLE_USER") && (
+            <>
+              <Route exact path="/collegiatesubreddits/list" element={<CollegiateSubredditsIndexPage />} />
+            </>
+          )
+        }
+        {
+          hasRole(currentUser, "ROLE_ADMIN") && (
+            <>
+              <Route exact path="/collegiatesubreddits/create" element={<CollegiateSubredditsCreatePage />} />
```

Comment:
The routes should be named the same casing as the API routes
```suggestion
              <Route exact path="/collegiateSubreddits/list" element={<CollegiateSubredditsIndexPage />} />
            </>
          )
        }
        {
          hasRole(currentUser, "ROLE_ADMIN") && (
            <>
              <Route exact path="/collegiateSubreddits/create" element={<CollegiateSubredditsCreatePage />} />
```